app/src/common/shared/com/igalia/wolvic/utils/DeviceType.java

```diff
@@ -53,6 +54,9 @@ public static void setType(@Type int aType) {
             case PicoG2:
                 name = "Pico G2";
                 break;
+            case PicoOXR:
+                name = "Pico OXR";
+                break;
```

Comment:
"Pico OXR" -> "Pico XR"

Also do please add a new isPicoXR() method by the very end of this file as well.